serial.md

```diff
@@ -166,6 +171,22 @@ other platforms.
 ```
 0  START_SYSEX        (0xF0)
 1  SERIAL_DATA        (0x60)
-2  SERIAL_LISTEN      (0x70) // OR with port to switch to (0x79 = switch to SW_SERIAL1)
-3  END_SYSEX          (0xF7)
+2  SERIAL_LISTEN      (0x70)
+3  portId             (any of the defined software serial IDs such as SW_SERIAL0)
+4  END_SYSEX          (0xF7)
+```
+
+### Serial Update Baud
+
+Update the baud rate on a configured serial port.
+
+```
+0  START_SYSEX        (0xF0)
+1  SERIAL_MESSAGE     (0x60)
+2  SERIAL_UPDATE_BAUD (0x01)
```

Comment:
At this point I either need to start back at 0x00 or 0x01 or start at 0x71 because 0x7F is the highest allowed value.